react-frontend/src/components/LoginComponent.js

```diff
@@ -0,0 +1,112 @@
+import React, { useState } from "react";
+import "bootstrap/dist/css/bootstrap.min.css";
```

Comment:
Please remove it.

1. Individual components should not import bootstrap CSS.
2. We have already imported bootstrap CSS at a common place, hence it's not required.